cw_evm/lib/evm_chain_wallet.dart

```diff
@@ -245,12 +245,11 @@ abstract class EVMChainWalletBase
     final outputs = _credentials.outputs;
     final hasMultiDestination = outputs.length > 1;
 
-    final String? opReturnMemo = outputs.first.memo;
+    final String? memo = outputs.first.memo;
 
     String? hexOpReturnMemo;
-    if (opReturnMemo != null) {
-      hexOpReturnMemo =
-          '0x${opReturnMemo.codeUnits.map((char) => char.toRadixString(16).padLeft(2, '0')).join()}';
+    if (memo != null) {
+      hexOpReturnMemo = '0x${memo.codeUnits.map((char) => char.toRadixString(16).padLeft(2, '0')).join()}';
```

Comment:
I don't think that this is needed anymore.
isn't this the previous implementation when we thought the memo would be sent in the `data` field, but now we can skip passing the `data` parameter at all

Reply:
Swaps for ETH don't work without the memo/data parameter. Transactions don't arrive at the destination address, so I had to include the data parameter again.